src/components/Seat/JetsSeat.test.js

```diff
@@ -0,0 +1,448 @@
+import { render, screen } from '@testing-library/react';
+
+import { MockJetsContextProvider } from '../../__mocks__/MockJetsContext';
+
+import {
+  paramsData,
+  seatDataFirst,
+  seatDataBusiness,
+  seatDataPremium,
+  seatDataEconomy,
+  seatDataAisle,
+  seatDataEmpty,
+  seatDataIndex,
+} from './__fixtures__';
+import { JetsSeat } from './index';
+
+const setup = ({ data = {}, config = {}, params = {} } = {}) => ({
+  ...render(
+    <MockJetsContextProvider config={config} params={{ ...paramsData(params) }}>
+      <JetsSeat data={data} />
+    </MockJetsContextProvider>
+  ),
+});
+
+describe('JetsSeat', () => {
+  describe('when any seat type is rendered', () => {
+    it('should add the correct classes when available', async () => {
+      setup({ data: seatDataFirst() });
+
+      const wrapper = screen.getByTestId('jets-seat');
+
+      // jets-seat twice due to component naming and seatType data
+      expect(wrapper).toHaveClass('jets-seat jets-seat jets-available');
+    });
+
+    it('should add the correct classes when unavailable', async () => {
```

Comment:
Why the need to declare the test as `async`?

Reply:
That's an oversight on my part. My initial implementation was awaiting child items in the render - I removed that but missed the `async`.